app/controllers/home_controller.rb

```diff
@@ -3,4 +3,13 @@ def index
     @chapters = Chapter.active.visitable.for_display.all
     @projects = Project.recent_winners.limit(15).all
   end
+
+  def feed
+    project_filter = ProjectFilter.new(Project.recent_winners)
+    @projects = project_filter.page(params[:page], 50).result
```

Comment:
We're going to want some eager loading of images here, otherwise we're going to be doing 50 db calls when we go and display the images in the output.